Content/Bosses/GlassMiniboss/Projectiles.GlassHammer.cs

```diff
@@ -179,13 +189,21 @@ public override void SetDefaults()
 		}
 
 		public override void OnSpawn(IEntitySource source)
+		{
+			OnSpawnSpikeVisualVars();
+		}
+
+		private void OnSpawnSpikeVisualVars()
 		{
 			Projectile.rotation += Main.rand.NextFloat(-0.01f, 0.01f);
 			maxSpikes = 3 + Main.rand.Next(16, 18);
 			points = new Vector2[maxSpikes];
 			offsets = new float[maxSpikes];
 			for (int i = 0; i < maxSpikes; i++)
```

Comment:
for loop needs closing braces, know this is unrelated but best to catch it while we can